src/extension/portmap/portmap.c

```diff
@@ -25,9 +25,6 @@ typedef struct Config {
 } Config;
 
 
-int valid_port_to_change(uint16_t port) {
-    return htons(port) > 1024;
```

Comment:
Hi,
Are you sure about removing this?
Without it, it means event ports already at 0 would also go through the network cooperation, which wouldn't be useful.